src/main/java/frc/robot/RobotContainer.java

```diff
@@ -569,6 +582,14 @@ private void configureVisionCommands() {
                 Commands.runOnce(drivetrain::resetPoseRotationToGyro)));
   }
 
+  private void configureShooterCommands() {
+    NoteVisualizer.setRobotPoseSupplier(this.drivetrain::getPose);
+    oi.getShootButton()
+        .onTrue(
+            Commands.parallel(
+                Commands.runOnce(() -> shooter.shoot(0.0, 0.0)), NoteVisualizer.shoot()));
+  }
+
```

Comment:
add all the other shooter bindings (aim, podium, amp, subwoofer, enable automated shooter)